packages/next/src/server/app-render/app-render.tsx

```diff
@@ -737,9 +737,17 @@ async function renderToHTMLOrFlightImpl(
         hasPostponed,
       })
 
+      function onHeaders(headers: Headers): void {
+        // Copy headers created by React into the response object.
+        headers.forEach((value: string, key: string) => {
```

Comment:
This will only actually be the `Link:` header so far but we could expand it later.